python/src/WeightedExperimentImplementation_doc.i.in

```diff
@@ -9,47 +9,77 @@ Parameters
 
-A WeightedExperiment object can be created only through its derived classes
-which are distributed in three groups:
+.. math::
 
-1. The first category is made up of the random patterns, where the set of input
-   data is generated from the joint distribution of the input random vector,
-   according to these sampling techniques:
+    \Expect{ \model(\physicalInputRandomVector)}
+    = \int_{\physicalInputSpace} \model(\physicalInputObservation) 
+    \inputProbabilityDensityFunction(\physicalInputObservation) d\physicalInputObservation
 
-   - :class:`Monte Carlo <openturns.MonteCarloExperiment>`
+where :math:`\inputProbabilityDensityFunction` is the probability density 
+function of the input random vector.
 
-   - :class:`LHS <openturns.LHSExperiment>`
+The class computes the weights :math:`\{w_i \in \Rset\}_{i = 1, ..., \sampleSize}`
+and the nodes :math:`\{\physicalInputObservation_i \in \Rset^{\physicalInputDimension}\}_{i = 1, ..., \sampleSize}`
+such that 
 
-   - :class:`Bootstrap <openturns.BootstrapExperiment>`
+.. math::
 
-   - :class:`Importance Sampling <openturns.ImportanceSamplingExperiment>`
+    \Expect{ \model(\physicalInputRandomVector)}
+    \approx \sum_{i = 1}^n w_i \model\left(\physicalInputObservation_i\right)
 
-2. The second category contains the :class:`low discrepancy sequences
-   <openturns.LowDiscrepancySequence>`. OpenTURNS proposes the Faure, Halton,
-   Haselgrove, Reverse Halton and Sobol sequences.
+By default, all the weights are equal to :math:`w_i = \frac{1}{n}`.
+Some quadrature rules e.g. the tensorised Gauss product rule guarantee 
+that all nodes are non negative, but not all rules have that 
+property.
 
-3. The third category consists of deterministic patterns:
+A `WeightedExperiment` object can be created only through its derived classes
+which are distributed in three groups:
 
-   - :class:`Gauss product <openturns.GaussProductExperiment>`
+1. The random patterns, where the set of input
+   data is generated from the joint distribution of the input random vector,
+   according to these sampling techniques e.g. 
+   :class:`Monte Carlo <openturns.MonteCarloExperiment>`.
+   In this case, the value returned by :meth:`~openturns.WeightedExperiment.isRandom()` is `True`.
 
-   - :class:`~openturns.FixedExperiment`
+2. The :class:`low discrepancy sequences
+   <openturns.LowDiscrepancySequence>` e.g. 
+   :class:`~openturns.SobolSequence`.
+   In this case, the value returned by :meth:`~openturns.WeightedExperiment.isRandom()` is `False`.
 
-   - :class:`~openturns.LowDiscrepancyExperiment`"
+3. The deterministic patterns or *quadrature rules* e.g.
+   :class:`Gauss product <openturns.GaussProductExperiment>`.
+   In this case, the value returned by :meth:`~openturns.WeightedExperiment.isRandom()` is `False`.
+
+In order to actually use a quadrature rule on a particular function,
+we suggest to use the :class:`~openturns.ExperimentIntegration` class."
```

Comment:
```suggestion
   In this case, the value returned by :meth:`~openturns.WeightedExperiment.isRandom()` is `False`.

To apply a quadrature rule (or any other design) on a particular function,
use the :class:`~openturns.ExperimentIntegration` class."
```
Otherwise you give the impression ExperimentIntegration only works with a deterministic pattern.

Reply:
That is true because the word "quadrature" has been introduced only for the third type of DoE. In this case, your suggestion does not fit neither. I suggest another sentence.

Reply:
Fixed at 77ef1abc2944c9ace2173f824e11c259d894d5a5. @josephmure : Please tell me if that fits.

Reply:
Great!